src/validator/MaintainValidator.cpp

```diff
@@ -323,6 +348,12 @@ Status CreateEdgeIndexValidator::validateImpl() {
     index_ = *sentence->indexName();
     fields_ = sentence->fields();
     ifNotExist_ = sentence->isIfNotExist();
+    if (sentence->comment() != nullptr) {
+        if (sentence->comment()->size() > SchemaUtil::kCommentLengthLimit) {
+            return Status::Error("Too long comment reach %ld bytes limit.",
+                                 SchemaUtil::kCommentLengthLimit);
+        }
+    }
```

Comment:
Why not check the length in Sentence related structure such that simplify the validator logics ? for example, do this in `setComment` ?

Reply:
Just want to threw it as SemanticError.

Reply:
Now the different sentence use the different struct for comment, so it's difficult to check it in one place.